hardhat.config.ts

```diff
@@ -73,6 +73,7 @@ const config: HardhatUserConfig = {
     },
     hardhat: {
       chainId: 1,
+      allowUnlimitedContractSize: true,
```

Comment:
what is the reason for activating this param?

Reply:
this was a test, I forget to remove it, I think there was a version of the contract that was too large, but it's now ok as it's deployed.